README.md

```diff
@@ -122,19 +122,19 @@ DPI=96
 Basic request that will search data and style in the prefix directory (see configuration)
 
 ```
-GET /rendergeojson?GEOJSON=data.json&STYLE=style.qml&BBOX=2689634.3,1283792.7,2690062.1,1284004.8&WIDTH=606&HEIGHT=300&DPI=96
+GET /ogc/?SERVICE=RenderGeojson&GEOJSON=data.json&STYLE=style.qml&BBOX=2689634.3,1283792.7,2690062.1,1284004.8&WIDTH=606&HEIGHT=300&DPI=96
```

Comment:
I don't expect the `ogc` prefix is widely used except for the openquake image. Maybe better replace it with `qgis_mapserv.fcgi?SERVICE=...`

Reply:
agreed, the important part is SERVICE=RenderGeojson
you just have ogc in an example further up so I used the same